### PR TITLE
Regen for Gtk 3.22.26

### PR DESCRIPTION
**Pull request by EPashkin**
Apply [last gir-files update](https://github.com/gtk-rs/gir-files/pull/14)

cc @GuillaumeGomez , @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

**Comment by EPashkin** [comment]
There possible problem with new functions and enum cases: it don't have version too.
Maybe I need add it?

**Comment by GuillaumeGomez** [comment]
Their version isn't generated? Is the information missing in the gir files or is gir failing to generate the cfg condition?

File: src/auto/flags.rs [hunk 1]
```diff
@@ -428,6 +428,8 @@ bitflags! {
         const UPPERCASE_SENTENCES = 64;
         const INHIBIT_OSK = 128;
         const VERTICAL_WRITING = 256;
+        const EMOJI = 512;
+        const NO_EMOJI = 1024;
```
Comment (EPashkin) [inline src/auto/flags.rs]: This added without version in Gtk-3.0.gir

File: src/auto/text_view.rs [hunk 2]
```diff
@@ -909,6 +913,18 @@ impl<O: IsA<TextView> + IsA<glib::object::Object> + glib::object::ObjectExt> Tex
...
+        unsafe {
+            let f: Box_<Box_<Fn(&Self) + 'static>> = Box_::new(Box_::new(f));
+            connect(self.to_glib_none().0, "insert-emoji",
+                transmute(insert_emoji_trampoline::<Self> as usize), Box_::into_raw(f) as *mut _)
+        }
+    }
+
+    fn emit_insert_emoji(&self) {
+        let _ = self.emit("insert-emoji", &[]).unwrap();
+    }
```
Comment (EPashkin) [inline src/auto/text_view.rs]: This too

**Comment by EPashkin** [comment]
Added i32 to ResponseType and IconSize conversion

**Comment by EPashkin** [comment]
@GuillaumeGomez These same problem in bitfield gdk::WindowState `TOP_TILED` and other 7 added without version in Gdk-3.0.gir

**Comment by GuillaumeGomez** [comment]
So I suppose we need a manual fix. Should we add a `version = "x.x.x"` in the toml file?

**Comment by EPashkin** [comment]
Updated as separate commit

**Comment by EPashkin** [comment]
Don't know how to do with gdk part: source says "GDK_WINDOW_STATE_TOP_TILED: whether the top edge is tiled, Since 3.91.2"
Version too big. As I remember it beta for 4.0 but this IMHO sill not published.

**Comment by GuillaumeGomez** [comment]
Let's just ignore it then. Kinda weird though.

**Comment by GuillaumeGomez** [comment]
Ok, now seems good to me. Thanks @EPashkin!

**Pull request merged**